tests/purchase_test.ts

```diff
@@ -0,0 +1,39 @@
+import LandingPage from "../pages/landingPage";
+import LoginPage from "../pages/loginPage";
+import ProductPage from "../pages/productPage";
+import CartPage from "../pages/cartPage";
+import CheckoutPage from "../pages/checkoutPage";
+import { User } from "../helpers/interfaces";
+import Products from "../testdata/products";
+
+Feature('Purchase');
+
+let products = new DataTable(['id', 'color', 'size']);
+products.add(['49', 'Green', 'Medium']);
```

Comment:
I like idea with id, but you should ask yourself, did I solve a problem or requirement with it? Did I make my code more simple? Keep id, but keep these questions in mindas well.

Reply:
hehe okay, I just thought it would be clumsy and unreadable to store all the urls, but apparently it is just me problem :'D

Reply:
I'd better delete Products at all